native/cocos/editor-support/spine-wasm/spine-type-export.cpp

```diff
@@ -646,39 +774,52 @@ EMSCRIPTEN_BINDINGS(spine) {
 
     class_<Skin>("Skin")
         .constructor<const String &>()
-        .function("getName", optional_override([](Skin &obj) { return STRING_SP2STD(obj.getName()); }))
-        .function("getBones", optional_override([](Skin &obj) { return VECTOR_SP2STD(obj.getBones()); }), allow_raw_pointers())
-        .function("getConstraints", optional_override([](Skin &obj) { return VECTOR_SP2STD(obj.getConstraints()); }), allow_raw_pointers())
-        .function("setAttachment", optional_override([](Skin &obj, size_t index, const std::string &name, Attachment *attachment) { return obj.setAttachment(index, STRING_STD2SP(name), attachment); }), allow_raw_pointers())
+        .function("getName", optional_override([](Skin &obj) {
+            return STRING_SP2STD(obj.getName()); }))
+        .function("getBones", optional_override([](Skin &obj) {
+            return VECTOR_SP2STD(obj.getBones()); }), allow_raw_pointers())
+        .function("getConstraints", optional_override([](Skin &obj) {
+            return VECTOR_SP2STD(obj.getConstraints()); }), allow_raw_pointers())
+        .function("setAttachment", optional_override([](Skin &obj, size_t index,
+        const std::string &name, Attachment *attachment) {
+            return obj.setAttachment(index, STRING_STD2SP(name), attachment);
+        }), allow_raw_pointers())
         .function("addSkin", select_overload<void(Skin *)>(&Skin::addSkin), allow_raw_pointers())
         .function("copySkin", select_overload<void(Skin *)>(&Skin::copySkin), allow_raw_pointers())
-        .function("getAttachment", optional_override([](Skin &obj, size_t slotIndex, const std::string &name) {
-                      return obj.getAttachment(slotIndex, STRING_STD2SP(name));
-                  }),
-                  allow_raw_pointers())
+        .function("findNamesForSlot", optional_override([](Skin &obj, size_t slotIndex,
+        std::vector<std::string> *names){
+            auto vetNames = new Vector<String>();
```

Comment:
Why to new `Vector<String>` ? Where to release its memory?

Could we just use `Vector<String> vetNames;` here?